vsifiwe/Papel
# #171227030 Hamburger menu

What does this PR do?
This PR shows a hamburger menu on small devices instead of showing a nav-bar

Description of Task to be completed?
Show an icon when testing responsiveness

How should this be manually tested?

-Inspect any page
-check responsiveness
-check for an icon (hamburger menu)
Any background context you want to provide?
This PR was based on received feedback

What are the relevant pivotal tracker stories?
#171227030
